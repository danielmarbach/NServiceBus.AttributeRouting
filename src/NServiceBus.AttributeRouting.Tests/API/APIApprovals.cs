using ApprovalTests;
using ApprovalTests.Reporters;
using PublicApiGenerator;
using System.Runtime.CompilerServices;
using Xunit;

namespace NServiceBus.AttributeRouting.Tests.API
{
    public class APIApprovals
    {
        [Fact]
        [UseReporter(typeof(DiffReporter))]
        [MethodImpl(MethodImplOptions.NoInlining)]
        public void Approve_API()
        {
            var publicApi = ApiGenerator.GeneratePublicApi(typeof(RouteToAttribute).Assembly);

            Approvals.Verify(publicApi);
        }
    }
}
