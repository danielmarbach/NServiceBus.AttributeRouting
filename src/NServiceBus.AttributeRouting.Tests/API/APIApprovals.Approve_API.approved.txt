[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETStandard,Version=v2.0", FrameworkDisplayName="")]
namespace NServiceBus.AttributeRouting
{
    public class AttributeRoutingFeature : NServiceBus.Features.Feature
    {
        public AttributeRoutingFeature() { }
        protected override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
    }
}
namespace NServiceBus
{
    public class static RoutingSettingsExtensions
    {
        public static void EnableAttributeRouting(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
    }
}